swagger.yml

```diff
@@ -0,0 +1,443 @@
+                $ref: "#/components/schemas/ErrorResponse"
+        "422":
+          description: Ошибка валидации
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+
+  /api/bnpl/general/v2/payment/state:
+    get:
+      tags:
+        - BNPL API
+      summary: Получить состояние.
+      description: Получить состояние платежа по ключу инедепонтенсности из ядра (CLICK core DB)
+      parameters:
+        - name: externalId
+          in: query
+          required: true
+          schema:
+            type: string
+            description: Уникальный ключ инедепонтенсности поставщика BNPL.
+            example: "eyJhbGciOiJIUzI1NiIsInR5.."
+      responses:
+        "200":
+          description: Успешный ответ
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  paymentId:
+                    type: integer
+                    description: Уникальный ID платежа.
+                    example: 3942185724
+                  status:
+                    type: integer
+                    description: Статус платежа.
+                    example: 2
+                  statusDescription:
+                    type: string
+                    description: Описание статуса.
+                    example: "Успешно"
+        "400":
+          description: Если платеж не найден
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+        "422":
+          description: Ошибка валидации
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+
+
+components:
+  securitySchemes:
+    OAuth2:
+      type: oauth2
+      flows:
+        clientCredentials:
+          tokenUrl: https://sandbox-bnpl.click.uz:8000/oauth/token
+          scopes: {}
+
+  schemas:
+    CreatePaymentRequest:
+      type: object
+      required:
+        - client_id
+        - amount
+        - externalId
+        - checkoutId
+      properties:
+        client_id:
+          type: number
+          description: "Идентификатор клиента в системе клик"
+          format: integer
+          example: 100001
+        amount:
+          type: number
+          description: "Сумма платежа"
+          format: float
+          example: 587.00
+        externalId:
+          type: string
+          description: "Идентификатор платежа. Сгенерирует внешная система"
+          format: uuid
+          example: "d9892e31-6669-0000-0000-0c8f34aa70f8"
+        checkoutId:
+          type: string
+          description: "Единый идентификатор для платежей"
+          format: uuid
+          example: "d9892e31-0000-4225-ab03-0c8f34bb70f8"
+
+    CreatePaymentResponse:
+      type: object
+      properties:
+        paymentId:
+          type: integer
```

Comment:
bit int cause we already have large value, it is already up to then general integer.